app/views/whats_new/index.html.erb

```diff
@@ -23,7 +23,28 @@
 
     <section class="app-view-whats-new__section">
       <h2 class="govuk-heading-l govuk-!-margin-bottom-3">Recent changes</h2>
-      <div>
+      <div class="govuk-!-static-margin-top-8">
+        <div class="govuk-grid-row">
+          <div class="govuk-grid-column-three-quarters">
+            <h3 class="govuk-heading-m">Move to GOV.UK design system</h3>
+          </div>
+          <div class="govuk-grid-column-one-quarter">
+            <strong class="govuk-tag govuk-tag--blue">
+              improvement
+            </strong>
+          </div>
+        </div>
+
+        <p class="govuk-body app-view-whats-new__last-updated">8 Sep 2023</p>
+
+        <p class="govuk-body">You will start seeing pages on Manuals Publisher moving to the GOV.UK design system.
+          This will make Manuals Publisher easier to navigate and consistent with other GOV.UK tools.
+          As the updates will be rolled out incrementally, you will notice some pages in the new design system and some
+          in the old design language. Please be assured that this is normal and all the pages will move to the GOV.UK
+          design system eventually.
+        </p>
```

Comment:
```suggestion
        <p class="govuk-body">You will start seeing pages on Manuals Publisher moving to the GOV.UK Design System.
          This will make Manuals Publisher easier to navigate and consistent with other GOV.UK tools.
          As the updates will be rolled out incrementally, you will notice some pages in the design while some remain
          in the old design. Please be assured that this is normal and all the pages will move to the GOV.UK
          Design System eventually.
        </p>
```

Reply:
Thanks Ollie!
The second suggested change reads a bit odd to me: `...As the updates will be rolled out incrementally, you will notice some pages in the design while some remain in the old design.`
Should it read `...As the updates will be rolled out incrementally, you will notice some pages in the new design while some remain in the old design.`

Reply:
Actually, having reread this in the cold light of morning, your text does make sense 😆 .

Reply:
Good spot - I honestly have no strong opinion either way! Go with whichever you prefer. 😄